wordpress-mobile/gutenberg-mobile
# Fix VideoPress block test cases

This PR aims to fix the failures in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5874 related to unit tests.

⚠️  It's expected that some of the CI jobs fails due to the RN upgrade. The jobs that this PR aims to fix are:
- `ci/circleci: Test Android`
- `ci/circleci: Test iOS`

We could omit the rest as they will be solved in other PRs.

**To test:**
* Run the command: `npm run test`.
* Observe that all tests pass.

Alternatively, we could rely on the CI job that runs the unit tests.

PR submission checklist:
